### PR TITLE
feat: Remove unused settings from activities and sequences [PT-184917932]

### PR DESCRIPTION
**Pull request by dougmartin**

### CODE REVIEW

File: spec/models/sequence_spec.rb [hunk 1]
```diff
@@ -171,80 +127,12 @@
...
-      expect(sequence_with_activities.serialize_for_portal("http://test.host")).to eq(complex_portal_hash)
-    end
-
     it 'returns a simple hash for an Activity Player sequence that can be consumed by the Portal' do
       expect(activity_player_sequence.serialize_for_portal('http://test.host')).to eq(ap_simple_portal_hash)
     end
 
   end
 
-  describe '#serialize_for_report_service' do
```
Comment (scytacki) [inline spec/models/sequence_spec.rb:188]: Doesn't this still exist? LARA still needs to send the activity structure to the report service in the report-service format.

**Review by scytacki — CHANGES_REQUESTED** [review]
This looks great.

I think I found a couple of things to address. 
You could ignore the `ENV` comment if you want, that is just a weird inconsistency.

File: app/models/sequence.rb [hunk 2]
```diff
@@ -119,28 +114,18 @@ def export(host)
...
-    else
-      run_url = local_url
-      source_type = "LARA"
-      append_auth_token = false
-      tool_id = ""
-    end
+    run_url = ap_url
+    source_type = "Activity Player"
+    append_auth_token = true
+    tool_id = ENV["ACTIVITY_PLAYER_URL"]
```
[the remark this answers is not in the document]
Comment (dougmartin) [reply app/models/sequence.rb:128]: I'm going to leave this as is :)

File: app/views/plugins/_author.haml [hunk 3]
```diff
@@ -12,12 +12,6 @@
   - firebase_jwt_url = api_v1_get_firebase_jwt_url({firebase_app: '_FIREBASE_APP_'})
 
   -# load runtime theme for preview of wrapped embeddable
```
[the remark this answers is not in the document]
Comment (dougmartin) [reply app/views/plugins/_author.haml]: Fixed in https://github.com/concord-consortium/lara/pull/1074/commits/4d0c77235530e1302f0d8d54f3c239569437ddb6

File: spec/models/sequence_spec.rb (hunk 1 above)
Comment (dougmartin) [reply spec/models/sequence_spec.rb]: Yes, added back (and updated to add preview_url) in https://github.com/concord-consortium/lara/pull/1074/commits/4d0c77235530e1302f0d8d54f3c239569437ddb6

**Review by scytacki — APPROVED** [review]
Looks good

**Pull request merged**